### ISSUE TITLE
ubuntu documentation site not working NVDA Firefox

### ISSUE THREAD

**Issue by abletec** [canonical-web-and-design/ubuntu.com#9785]
NVDA is a Windows screen reader used by blind individuals to read what's on screen. It appears your site has some sort of cookie banner that NVDA does not read in Firefox. I was able to access the site using Chrome, but it's really a hassle having to switch browsers, especially when this is not a terribly difficult problem to fix. Thanks.

---
*Reported from: https://ubuntu.com/server/docs/install/storage*

**Comment by anthonydillon** [comment]
Thank you for taking the time to report this issue. I will look into it today.

**Comment by anthonydillon** [comment]
I believe this is related to this issue: https://github.com/nvaccess/nvda/issues/2745. I have replicated the behaviour in both Firefox which does not work and Chrome which does. I believe the markup should read the landmark as the region is focused but I am no expert. 

Does anyone else have any other suggestions?

**Comment by abletec** [comment]
@anthonydillon, this has something to do w/the way this alert is being coded, methinks. If it'd just be changed to a simple dialog w/an "ok" or "got it" button or equivalent, I think everything would be fine.

**Comment by sowasred2012** [comment]
@abletec I believe this problem was fixed by another PR (https://github.com/canonical-web-and-design/cookie-policy/pull/123) in July - I've just tested the site with NVDA, and from what I can tell it seems to be OK, but please let us know if not.

**Comment by abletec** [comment]
Gentlemen, the cookie dialog now can be easily accessed by NVDA, so I
can now visit the site. Thanks so much for that. Nice to be taken
seriously when these sorts of bugs are reported.

On 12/3/21, Scott Mason Nash ***@***.***> wrote:
> @abletec I believe this problem was fixed by another PR
> (https://github.com/canonical-web-and-design/cookie-policy/pull/123) in July
> - I've just tested the site with NVDA, and from what I can tell it seems to
> be OK, but please let us know if not.
>
> --
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/canonical-web-and-design/ubuntu.com/issues/9785#issuecomment-985916711


-- 
Subscribe to a WordPress for Newbies Mailing List by sending a message to:
***@***.*** with 'subscribe' in the Subject field OR by
visiting the list page at http://www.freelists.org/list/wp4newbs
& check out my sites at www.brightstarsweb.com & www.mysitesbeenhacked.com

**Comment by sowasred2012** [comment]
@abletec not a problem! I'll close this issue, but if you encounter other problems of this nature, please let us know.
